### PR TITLE
Add ALmixer.

### PR DESCRIPTION
**Pull request by Rinnegatamante**

### CODE REVIEW

File: ALmixer/VITABUILD [hunk 1]
```diff
@@ -0,0 +1,25 @@
+pkgname=ALmixer
+pkgver=9999
+pkgrel=1
+url="https://github.com/Rinnegatamante"
```
Comment (isage) [inline ALmixer/VITABUILD:4]: Wrong url.

File: ALmixer/VITABUILD [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+build() {
+  cd $pkgname
+  mkdir build && cd build
+  cmake .. -DVITA=ON
+  make -j$(nproc)
+}
+
+package () {
+  mkdir -p $pkgdir/$VITASDK/arm-vita-eabi/lib
```
Comment (isage) [inline ALmixer/VITABUILD:17]: Cmake had install step, is `make install ` not enough?

**Comment by isage** [comment]
Does it really needs to be a fork? Upstream+patch seems better, with fixed (0.5.0) version.

**Comment by Rinnegatamante** [comment]
Upstream is dead (the original repo was on bitbucket) and couldn't find an official still maintained mirror anywhere which is why i ended up just forking it for sake of simplicity.

**Comment by isage** [comment]
Fair.

**Comment by isage** [comment]
Still, can you use fixed version instead of 9999 and target specific commit (or maybe tag it and target tag)?

File: ALmixer/VITABUILD (hunk 2 above)
Comment (isage) [reply ALmixer/VITABUILD:17]: I guess i can take a look at that later (after merge).

**Comment by Rinnegatamante** [comment]
Done

File: ALmixer/VITABUILD [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+pkgver=0.5
+pkgrel=1
+url="https://github.com/Rinnegatamante/ALmixer"
+source=("https://github.com/Rinnegatamante/$pkgname/archive/$pkgver.tar.gz")
+sha256sums=('6750226427449c371f955dbbb217dc857f55b545d57b916a009c4e17cddbba86')
+depends=('libvorbis' 'mpg123' 'libogg')
+
+build() {
+  cd $pkgname-$pkgver
+  mkdir build && cd build
```
Comment (isage) [inline ALmixer/VITABUILD:11]: `-DCMAKE_BUILD_TYPE=Release` ?

**Comment by isage** [comment]
Commit hash now needs an update

**Comment by isage** [comment]
looks good :+1:

**Pull request merged**